Tools/wpe/meson-wrapper

```diff
@@ -0,0 +1,14 @@
+#! /bin/sh
+set -e
+
+reconfigure_mode=$1
+shift
+
+if [ "$reconfigure_mode" = flag ] || [ -r "$1/build.ninja" ] ; then
```

Comment:
I think in this statement variable `$1` can be replaced for `$reconfigure_mode`.